pkg/cmd/completion/completion.sh

```diff
@@ -9,6 +9,23 @@ __dcos_debug() {
     fi
 }
 
+
+# checks if an array of subcommands contains a subcommand.
+# usage: _contains_subcommand <arrays_of_subcommands> <cmd>
+_contains_subcommand () {
```

Comment:
I don't think this is too important but it doesn't follow the naming scheme for helper functions. I'm not sure how common of a name this might be but I figured it's better to namespace these under `__dcos` since these functions will be global within a shell.

Also, it seems like you're not using this function anywhere.

Reply:
Ok, I can update the naming scheme. 
Regarding the use of this function: As of now none of the functions where we have dynamic completions need this fix. Going forward we can use this function when adding more commands that support dynamic argument completion.